template.distillery/Makefile.style

```diff
@@ -27,7 +27,7 @@ $(CSSDIR):
 $(DEST): $(LOCAL_CSS) | $(CSSDIR)
 	HASH=`cat $< | md5sum | cut -d ' ' -f 1` && \
 	cp $< $(CSS_PREFIX)_$$HASH.css && \
-	sed -i '1s/^/@charset "UTF-8";/' $(CSS_PREFIX)_$$HASH.css && \
+	sed -i .css '1s/^/@charset "UTF-8";/' $(CSS_PREFIX)_$$HASH.css && \
```

Comment:
simplest way is probably to write `sed -i -e '1s/^/@charset "UTF-8";/' $(CSS_PREFIX)_$$HASH.css && \ `